nautobot/docs/configuration/authentication/sso.md

```diff
@@ -56,7 +56,7 @@ Please see the SAML configuration guide below for an example of how to configure
 
 To use external authentcation, you'll need to define `AUTHENTICATION_BACKENDS` in your `nautobot_config.py`.
```

Comment:
Since you're in here anyway, mind fixing this? :-)

```suggestion
To use external authentication, you'll need to define `AUTHENTICATION_BACKENDS` in your `nautobot_config.py`.
```